vagrant/k8s/Vagrantfile

```diff
@@ -84,14 +84,11 @@ echo "export no_proxy='k8master,192.168.2.10,192.168.2.11,192.168.2.12,192.168.2
 echo "export no_proxy='k8master,192.168.2.10,192.168.2.11,192.168.2.12,192.168.2.13,netmaster,localhost,127.0.0.1'" >> ~/.profile
 source /etc/profile.d/envvar.sh
 # Change ownership for gopath folder
-sudo chown -R vagrant #{gopath_folder}
-sudo yum install -y net-tools
-sudo yum install -y go
-sudo yum install -y git
+sudo chown -R vagrant:vagrant #{gopath_folder}
+sudo yum install -y net-tools git
 sudo -E bash
 go get github.com/tools/godep
-cd $GOPATH/src/github.com/tools
-go install ./godep
+sudo chown -R vagrant:vagrant #{gopath_folder}
```

Comment:
@DivyaVavili: I had to add the second one because the compilation was failing. I can remove the first one and keep the second if you'd prefer that.